include/rapidjson/reader.h

```diff
@@ -152,7 +152,8 @@ enum ParseFlag {
     kParseCommentsFlag = 32,        //!< Allow one-line (//) and multi-line (/**/) comments.
     kParseNumbersAsStringsFlag = 64,    //!< Parse all numbers (ints/doubles) as strings.
     kParseTrailingCommasFlag = 128, //!< Allow trailing commas at the end of objects and arrays.
-    kParseNanAndInfFlag = 256,      //!< Allow parsing NaN, Inf, Infinity, -Inf and -Infinity as doubles.
+	kParseNanAndInfFlag = 256,      //!< Allow parsing NaN, Inf, Infinity, -Inf and -Infinity as doubles.
+	kParseMultiLineValueFlag = 512, //!< Allow parsing multi-line string values
```

Comment:
weird indent

Reply:
```suggestion
	kParseMultiLineStringFlag = 512, //!< Allow parsing multi-line string values.
```
minor: better name for the intent of this flag?

also super minor: added a period at end of the code comment to match the other lines above it.

Reply:
The weird indent isn't showing up locally for me

Reply:
Indent looks fine to me too; I use a chrome plugin for GitHub that actually fixes the indent alignment to show accurately (whereas GitHub by default doesn't), give Refined GitHub a try if you'd like, I do recommend changing a couple of settings:

Disable the following in extension's Options:
- clean-conversation-headers (so that the target branch of a PR can still be seen)
- use-first-commit-message-for-new-prs (when otherwise in conflict with use of PR templates)

Reply:
Oh well actually, the lines above this are using spaces, whereas you changed line 155 to use a tab and the new line 156 uses a tab.